ros_ws/src/control/src/control/remote_control.cpp

```diff
@@ -17,7 +17,7 @@ RemoteControl::RemoteControl()
     cmd_vel.angular.y = 0;
     cmd_vel.angular.z = 0;
     speed_modifier = 0.5;
-
+    config = initConfigs();
```

Comment:
I would separate this into its own code block to distinguish it from the rest of the initialization.  It might also be helpful if you left a comment describing what it does here so its not necessary to go looking for its implementation later.